tracer/src/Datadog.Trace/IAST/EvidenceConverter.cs

```diff
@@ -0,0 +1,240 @@
+// <copyright file="EvidenceConverter.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using Datadog.Trace.Iast.SensitiveData;
+using Datadog.Trace.Iast.Settings;
+using Datadog.Trace.Vendors.Newtonsoft.Json;
+using Datadog.Trace.Vendors.Newtonsoft.Json.Utilities;
+
+namespace Datadog.Trace.Iast;
+
+internal class EvidenceConverter : JsonConverter<Evidence>
+{
+    private bool _redactionEnabled;
+
+    public EvidenceConverter(bool redactionEnabled)
+    {
+        _redactionEnabled = redactionEnabled;
+    }
+
+    public override bool CanRead => false;
+
+    public override Evidence ReadJson(JsonReader reader, Type objectType, Evidence existingValue, bool hasExistingValue, JsonSerializer serializer)
+    {
+        throw new NotImplementedException();
```

Comment:
I guess that this method can be removed.

Reply:
Actually not, because it is abstract in the base class